models/migrations/v145.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"code.gitea.io/gitea/modules/timeutil"
+	"xorm.io/xorm"
+)
+
+func increaseLanguageField(x *xorm.Engine) error {
+	// THIS MUST BE THE FULL DEFINITION FOR recreateTable TO WORK CORRECTLY
```

Comment:
Does it really need to recreate? Can we do the equivalent of `alter table <table> alter column <column> varchar(50)` maybe?

Reply:
If I could have made that work I would have.

Reply:
OK because we don't have to change the type of sqlite's column so we don't have to deal with its constraint changes, and because we don't have to deal with constraint changes in general we can simplify a lot.